### DISCUSSION TITLE
01/004-%E7%AC%AC%E5%9B%9B%E7%AB%A0-%E9%BA%BB%E7%BE%8E%E8%A7%82%E5%BD%B1%E8%AE%B0-%E4%B8%8B

### DISCUSSION THREAD

**Discussion by giscus** [liuli-moe/to-the-stars#61] [General]
# 01/004-%E7%AC%AC%E5%9B%9B%E7%AB%A0-%E9%BA%BB%E7%BE%8E%E8%A7%82%E5%BD%B1%E8%AE%B0-%E4%B8%8B

在经历了几个世纪的动荡之后，一个乌托邦式的 AI— 人类政府治理着地球，预示着后稀缺社会的来临和太空殖民的新时代。一次意外的接触却让科技更先进的敌对外星种族打破了和平，这迫使魔法少女们走出幕后，拯救人类文明。在这一切之中，志筑良子，一个普通的女孩，仰望着星空，好奇着她在宇宙中的归所。

https://tts.liuli.moe/01/004-%E7%AC%AC%E5%9B%9B%E7%AB%A0-%E9%BA%BB%E7%BE%8E%E8%A7%82%E5%BD%B1%E8%AE%B0-%E4%B8%8B.html

**Comment by USS-Raven** [comment]


**Reply by rxliuli** [reply]
似乎是个很古老的错误，你想提个 pr 修复它么？
